### PR TITLE
Add Jetpack Starter to pricing page

### PR DESCRIPTION
**Pull request by IanRamosC**
Related to pbNhbs-5V7-p2

## Proposed Changes

* This PR adds Jetpack Starter to cloud.jetpack.com/pricing as one of the main bundles offered. It removes code that is unused and follow the design proposed in 0MMYGZ7bRaHuwMF3fFS2VC-fi-3402_25235.

NOTE: not everything in the file above is implemented here as it would require a minor refactor, and it also overlaps with another project

## Testing Instructions

* Checkout this branch locally
* Run `yarn start` and `yarn start-jetpack-cloud-p`
* Visit jetpack.cloud.localhost:3001/pricing?calypso_env=development and make sure you see the new product and the design matches the one in Figma
* Go through the purchase process and make sure it works as expected
image
image

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by IanRamosC** [comment]
Sorry for the ping y'all. Accidentally changed the base to the other branch without rebasing it first. :|

**Comment by robertsreberski** [comment]
Just noting the price is not 50% off, as mentioned at the top of the page:
CleanShot 2023-05-05 at 10 43 56@2x

I don't think it's a big deal though, we can address it later.

File: client/my-sites/plans/jetpack-plans/product-store/hero-image/index.tsx [hunk 1]
```diff
@@ -1,97 +1,39 @@
...
-		md: CompleteMd,
-		sm: CompleteSm,
-	},
+const HERO_IMAGES: Record< string, string > = {
+	[ PLAN_JETPACK_STARTER_YEARLY ]: HeroStarter,
+	[ PLAN_JETPACK_STARTER_MONTHLY ]: HeroStarter,
+	[ PLAN_JETPACK_SECURITY_T1_YEARLY ]: HeroSecurity,
+	[ PLAN_JETPACK_SECURITY_T1_MONTHLY ]: HeroSecurity,
+	[ PLAN_JETPACK_COMPLETE ]: HeroComplete,
+	[ PLAN_JETPACK_COMPLETE_MONTHLY ]: HeroComplete,
```
Comment (robertsreberski) [inline client/my-sites/plans/jetpack-plans/product-store/hero-image/index.tsx:21]: Why do we remove different sizing of images?

File: client/my-sites/plans/jetpack-plans/product-store/items-list/style-most-popular.scss [hunk 2]
```diff
@@ -22,13 +22,13 @@
 	list-style-type: none;
 	display: grid;
 	grid-template-columns: 1fr;
-	grid-gap: 24px;
+	grid-gap: 30px;
```
Comment (robertsreberski) [inline client/my-sites/plans/jetpack-plans/product-store/items-list/style-most-popular.scss:25]: Is this gap related to the change in design? It seems counter-intuitive to add more columns in the row and increase the gap size between them. But it also makes sense if it makes the design more esthetic.

**Review by robertsreberski — APPROVED** [review]
The pricing page looks good after the changes. Besides minor comments, code looks good.

File: client/my-sites/plans/jetpack-plans/product-store/hero-image/index.tsx (hunk 1 above)
Comment (IanRamosC) [reply client/my-sites/plans/jetpack-plans/product-store/hero-image/index.tsx:21]: They were being used when we have the hero for products and bundles (now we only have bundles), and the card style was different based on the screen size (now we have the same style across different screen sizes)

File: client/my-sites/plans/jetpack-plans/product-store/items-list/style-most-popular.scss (hunk 2 above)
Comment (IanRamosC) [reply client/my-sites/plans/jetpack-plans/product-store/items-list/style-most-popular.scss:25]: Yep, it's related to the design change proposed. It looks counterintuitive, but the hero image is now at the top of the card instead of on the side - that's why the gap size can be bigger.

File: client/my-sites/plans/jetpack-plans/product-store/hero-image/index.tsx [hunk 3]
```diff
@@ -1,97 +1,39 @@
 import {
```
Comment (CodeyGuyDylan) [inline client/my-sites/plans/jetpack-plans/product-store/hero-image/index.tsx:1]: Nice cleanup in this file

**Review by CodeyGuyDylan — CHANGES_REQUESTED** [review]
I noticed one design discrepancy. It looks like all the links above the CTA should be aligned in figma
image
I also see this being exacerbated on tablet-sized screens, but I also see it on the Security card so it's not necessarily an issue with this PR
image

One more thing, the icon in the lightbox for Jetpack Starter looks a bit small to me
image

In the lightbox I am seeing Akismet show 10K calls per month when I believe it should be 1K
image

**Comment by CodeyGuyDylan** [comment]
One more thing, I see this section on the figma designs
image
Are we considering this to be not needed for i1 and outside the scope of this PR? I think I would agree with that, just confirming

**Comment by elliottprogrammer** [comment]
Also I noticed in the lightbox figma designs, design has added "**_callouts_**" labels, specific to the amount of storage and API calls (See images comparison below).

I'm guessing we could also consider this **out of scope** or i1 and this PR, and could be added as a follow-up or Maintenance task, right?.  What do you think?

This PR | Figma design
--- | ---
 | 

I know we want to ship this and get it in front of users asap, so I wouldn't consider this a blocker and mainly just posting so we can track it. 👍

**Comment by IanRamosC** [comment]
> I'm guessing we could also consider this **out of scope** or i1 and this PR, and could be added as a follow-up or Maintenance task, right?. What do you think?

Thanks for reviewing. Yep, that's gonna be added in a follow-up PR.

**Review by elliottprogrammer — APPROVED** [review]
Tests well and looks great!  There's a couple small additions out of scope of this PR, but we can handle these in follow-up PR's.
Overall LGTM! 👍

**Review by CodeyGuyDylan — APPROVED** [review]
This all LGTM, thanks for addressing those comments 😄

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7970226

Thank you @IanRamosC for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7970226).
